docs/src/operator/usage.md

```diff
@@ -32,6 +32,34 @@ miden-node bundled bootstrap \
   --accounts-directory .
 ```
 
+Alternatively, provide an explicit toml-formatted configuration which allows you to define
+multiple assets and faucets easily at chain startup.
+
+```sh
+miden-node bundled bootstrap \
+  --data-directory data \
+  --accounts-directory . \
+  --genesis-config-file genesis.toml
+```
+
+The genesis configuration file should contain at least one faucet, and optionally, wallet definitions with assets, for example:
```

Comment:
@PhilippGackstatter what happens if we insert assets into the genesis block which have no associated faucet account? Would that break, and/or not be possible?

@drahnr they could technically also create an empty genesis block - they technically don't need an official faucet. Or they could just define a bunch of normal accounts, though whether they can define assets without an associated faucet account is not clear to me yet ^^.

Reply:
The current API does not allow to have an asset without an associated faucet identifier, at least from the API we have today.
While I think that's technically correct, I don't think it's practically useful. So I'd rather not document that.